pkg/transport/transport.go

```diff
@@ -130,6 +138,15 @@ func (transport *Transport) WithObservers(obs ...Observer) Client {
 	return tr
 }
 
+func getIP(target string) (string, error) {
+	addrElements := strings.Split(target, ":")
```

Comment:
This is not the proper way to split an IP address. This will fail with ipv6

Reply:
I now see `net.SplitHostPort()`. Do you know if ipv6 targets will already have brackets around the host portion?

Reply:
https://en.wikipedia.org/wiki/IPv6#Addressing, but we probably can assume, yes.